File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/vt/UserRoles.java
package com.payhub.ws.vt;

import java.util.ArrayList;

import com.fasterxml.jackson.core.JsonProcessingException;
import com.fasterxml.jackson.databind.DeserializationFeature;
import com.fasterxml.jackson.databind.ObjectMapper;

public class UserRoles {
	
		public final static String ALL_USER_ROLE_LINK="userRole/roles";
		public ArrayList<Roles> userRoles;
		
		public UserRoles(){}
		
		
		/**
		 * @return the userRoles
		 */
		public ArrayList<Roles> getUserRoles() {
			return userRoles;
		}

		/**
		 * @param userRoles the userRoles to set
		 */
		public void setUserRoles(ArrayList<Roles> userRoles) {
			this.userRoles = userRoles;
		}


		public static class Roles{
			
			public String numberOfUsers;
			public String roleName;
			public String roleId;
			
			public Roles() {
			}
			/**
			 * @return the numberOfUsers
			 */
			public String getNumberOfUsers() {
				return numberOfUsers;
			}
			/**
			 * @param numberOfUsers the numberOfUsers to set
			 */
			public void setNumberOfUsers(String numberOfUsers) {
				this.numberOfUsers = numberOfUsers;
			}
			/**
			 * @return the roleName
			 */
			public String getRoleName() {
				return roleName;
			}
			/**
			 * @param roleName the roleName to set
			 */
			public void setRoleName(String roleName) {
				this.roleName = roleName;
			}
			/**
			 * @return the roleId
			 */
			public String getRoleId() {
				return roleId;
			}
			/**
			 * @param roleId the roleId to set
			 */
			public void setRoleId(String roleId) {
				this.roleId = roleId;
			}
			
			
		}
		
}

File: Payhub - SDK/php_SDK/com/payhub/ws/vt/UserRoles.php
<?php

/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 12/21/15
 * Time: 4:07 PM
 */
class UserRoles
{
    public static $ALL_USER_ROLE_LINK="userRole/roles";
    public $userRoles=[];

    /**
     * UserRoles constructor.
     */
    public function __construct()
    {
    }

    public static function fromArray($data)
    {
        $settings = new UserRoles();
        foreach ($data as $key => $value) {
            if (property_exists(get_class($settings), $key)) {
                if ($key == "userRoles") {
                    $_tmp = new Roles();
                    foreach ($value as $keys => $val) {
                        $_tmp = Roles::fromArray($val);
                        $myArray[] = $_tmp;
                        }
                    $settings->{$key} = $myArray;
                    //var_dump($myArray);

                    }

            }
        }
       // var_dump($settings);
        return $settings;
    }

    /**
     * @return array
     */
    public function getUserRoles()
    {
        return $this->userRoles;
    }

    /**
     * @param array $userRoles
     */
    public function setUserRoles($userRoles)
    {
        $this->userRoles = $userRoles;
    }

}

class Roles{
    public $numberOfUsers;
    public $roleName;
    public $roleId;
    /**
     * UserRoles constructor.
     */
    public function __construct()
    {
    }
    public static function fromArray($data)
    {

        $settings = new Roles();
        foreach ($data as $key => $value) {
            if (property_exists(get_class($settings), $key)) {
                $settings->{$key} = $value;
            }
        }
        return $settings;
    }

    /**
     * @return mixed
     */
    public function getNumberOfUsers()
    {
        return $this->numberOfUsers;
    }

    /**
     * @param mixed $numberOfUsers
     * @return Roles
     */
    public function setNumberOfUsers($numberOfUsers)
    {
        $this->numberOfUsers = $numberOfUsers;
        return $this;
    }

    /**
     * @return mixed
     */
    public function getRoleName()
    {
        return $this->roleName;
    }

    /**
     * @param mixed $roleName
     * @return Roles
     */
    public function setRoleName($roleName)
    {
        $this->roleName = $roleName;
        return $this;
    }

    /**
     * @return mixed
     */
    public function getRoleId()
    {
        return $this->roleId;
    }

    /**
     * @param mixed $roleId
     * @return Roles
     */
    public function setRoleId($roleId)
    {
        $this->roleId = $roleId;
        return $this;
    }



}